neuro_extras/main.py

```diff
@@ -257,28 +262,44 @@ async def _data_cp(
         raise ValueError(
             "This command can't be used to copy data between two storage locations"
         )
+    if source_url_type == UrlType.DISK and destination_url_type == UrlType.DISK:
+        raise ValueError(
+            "This command can't be used to copy data between two persistent disks"
+        )
+
+    # Persistent disk and storage locations must be mounted as folders to a job
+    if UrlType.STORAGE in (source_url_type, destination_url_type) or UrlType.DISK in (
+        source_url_type,
+        destination_url_type,
+    ):
+        if source_url_type == UrlType.STORAGE:
+            volume.append(f"{str(source_url)}:/var/storage")
```

Comment:
is it an error? `storage:` requires `:rw`/`:ro`, `disk:` doesn't

Reply:
`disk:` definitely does, see https://github.com/neuro-inc/platform-client-python/pull/1721/files#diff-5d4eb6fdbc1861e5c98e67d287c0016eR79

However looks like `storage` also supports `:rw`/`:ro`, but the code works without any modifier.